prow/plugins/trigger/generic-comment.go

```diff
@@ -183,3 +226,40 @@ func getContexts(combinedStatus *github.CombinedStatus) (sets.String, sets.Strin
 	}
 	return failedContexts, allContexts
 }
+
+func addHelpComment(githubClient githubClient, body, org, repo, branch string, number int, presubmits []config.Presubmit, HTMLURL, user, note string, logger *logrus.Entry) error {
+	available, err := availablePresubmits(githubClient, body, org, repo, branch, number, presubmits, logger)
+	if err != nil {
+		return err
+	}
+	var resp string
+	if len(available) > 0 {
+		resp = fmt.Sprintf("%sThe following commands are available to trigger jobs:%s\n\nUse `/test all` to run all jobs.",
+			note,
+			func(names []string) string {
```

Comment:
The inline func to be able to do that in the arg is a bit too fancy for my taste, can we just build the string after the `len(available) > 0 ` check?

Reply:
Thank you for the review. Fixed.